drivers/dai/intel/ssp/ssp.h

```diff
@@ -40,17 +40,10 @@
 #define DAI_INTEL_SSP_PLATFORM_DEFAULT_DELAY	12
 #define DAI_INTEL_SSP_DEFAULT_TRY_TIMES		8
 
-#if CONFIG_SOC_INTEL_CAVS_V15
 /** \brief Number of 'base' SSP ports available */
-#define DAI_INTEL_SSP_NUM_BASE			4
+#define DAI_INTEL_SSP_NUM_BASE		DT_INST_PROP_BY_PHANDLE(0, settings, ports_base)
```

Comment:
Store this in `struct dai_config`

Reply:
I'm also not sure I get why... while this configuration is also used in runtime, it's also used to define the size of an array and it seems fairly intel ssp specific - why would it belong to the generic struct?

Reply:
Fair point about being Intel SSP specific. Is there another way to avoid hard-coding the instance number?